Marlin/language_de.h

```diff
@@ -88,6 +93,7 @@
 #define MSG_PID_I                           "PID I"
 #define MSG_PID_D                           "PID D"
 #define MSG_PID_C                           "PID C"
+#define MSG_SELECT                          "Auswählen"
 #define MSG_ACC                             "A"
 #define MSG_VXY_JERK                        "V XY Jerk"
 #define MSG_VZ_JERK                         "V Z  Jerk"
```

Comment:
Eben dieser link is nicht hilfreich, weil 'jerk' in Marlin anders definiert ist. (http://forums.reprap.org/read.php?245,676970)

Eine Suche nach "Marlin jerk" war bisher immer wesentlich Zielführender.